File: Kotlin/time.kt
fun main() {
    while (true) {
        // clear screen
        print("\u001b[2J\u001b[H")
        val now = java.time.LocalTime.now()
        // format time HH:MM:SS
        val formatter = java.time.format.DateTimeFormatter.ofPattern("HH:mm:ss")
        // print current time
        println("Current time: ${now.format(formatter)}")
        // sleep for 1 second (1000 milliseconds)
        Thread.sleep(1000)
    }
}

File: Dart/time.dart
import 'dart:io';

void main() {
    while (true) {
        print("\x1B[2J\x1B[0;0H");
        var currentTime = DateTime.now();
        var currentHour = currentTime.hour;
        var currentMinute = currentTime.minute;
        var currentSecond = currentTime.second;
        print("Current time: $currentHour:$currentMinute:$currentSecond");
        sleep(Duration(seconds: 1));
        // clear the screen
    }
}